### PR TITLE
comment

### PR DESCRIPTION
**Pull request by bam241**
trying to add comment where I don't understand some sentences.

### CODE REVIEW

File: abstract.tex [hunk 1]
```diff
@@ -72,7 +72,7 @@
 \noindent\textbf{Abstract:} 
 
 The objective of this project is to develop an open and collaborative approach
-for establishing increased confidence in \gls{NFCS} tools. This approach will
+for establishing increased confidence in nuclear fuel cycle simulator\gls{NFCS} tools. This approach will
```
Comment (gonuke) [inline abstract.tex]: this uses the acronym capability to do this automatically

File: narrative/1_objective.tex [hunk 2]
```diff
@@ -22,7 +22,7 @@ \section{Project Objectives}
 technology.
 
 The development of this capability in an open and collaborative fashion will
-support innovation in this space, as it makes it possible for new entrants to
+support innovation in this space, as it makes possible for new entrants to
```
Comment (gonuke) [inline narrative/1_objective.tex]: the original grammar is correct... 
"make possible" needs a direct object

File: narrative/1_objective.tex [hunk 3]
```diff
@@ -12,8 +12,8 @@ \section{Project Objectives}
 that community.
 
 This kind of comparison will also build knowledge and understanding about the
-differences between the various algorithms and models used by the many fuel
-cycle simulation tools. In particular, it is important to understand the
+differences between the various algorithms and models used by the many
+NFCS tools. In particular, it is important to understand the
```
Comment (gonuke) [inline narrative/1_objective.tex]: use the acronym expander `\gls{NFCS}`

File: narrative/1_objective.tex [hunk 4]
```diff
@@ -42,6 +42,8 @@ \section{Project Objectives}
 campaign demonstrated how this kind of analysis might be used to better
 understand the transitions that would need to occur over the many-decade time
 scale to arrive at demonstrably improved nuclear fuel cycles.  The fidelity
+%I don't understand the last part ``at demonstrably improved nuclear fuel
+%cycles''
```
Comment (gonuke) [inline narrative/1_objective.tex]: nuclear fuel cycles that have been demonstrated to be improved

File: narrative/2_scope.tex [hunk 5]
```diff
@@ -67,7 +67,7 @@ \section{Proposed Scope}
 tool than another.  Given the very high-dimensional decision space for
 modeling future nuclear fuel cycles, different tools often focus on different
 subsets of that decision space.  Therefore, any given problem specification
-may full specify a scenario for one tool but be missing important information
+may fully specify a scenario for one tool but be missing important information
```
Comment (gonuke) [inline narrative/2_scope.tex:70]: :+1:

File: narrative/2_scope.tex [hunk 6]
```diff
@@ -77,7 +77,7 @@ \section{Proposed Scope}
 there are parts of decision space that are not validated, and little insight
 is gained regarding the domain of applicability of any given tool or its
 underlying models. \emph{This project will develop a set of problem
-  specifications that is designed to cover the decision space and completely
+  specifications that is designed to cover the decision space as completely
```
Comment (gonuke) [inline narrative/2_scope.tex:80]: :+1:

File: narrative/3_1_fcci.tex [hunk 7]
```diff
@@ -21,6 +21,7 @@ \subsection{\gls{FCCI}}
 emerged during two recent workshops focused on this specific community.  Early
 successes will be used to showcase the effort at conferences and workshops to
 promote broader participation.
+% Shall we include reference to those workshop ?
```
Comment (gonuke) [inline narrative/3_1_fcci.tex]: I mention it explicitly in the Schedule section - we could mention it earlier here, too.

File: narrative/3_1_fcci.tex [hunk 8]
```diff
@@ -18,7 +18,8 @@ \subsection{\gls{FCCI}}
 and users of \gls{NFCS} tools.  Given the number and national variety of these
 partners, direct funding of their participation is beyond the scope of this
 project.  However, sincere expressions of interest from many parties have
-emerged during two recent workshops focused on this specific community.  Early
+emerged during two recent workshops (Technical Workshops on Fuel Cycle
+Simulation, 2015 and 2016) focused on this specific community.  Early
```
Comment (bam241) [inline narrative/3_1_fcci.tex]: I did not add citations as I was not sure we can afford them....

**Review by bam241 — COMMENTED** [review]
@gonuke I am done with this.

File: narrative/3_1_fcci.tex (hunk 8 above)
Comment (bam241) [reply narrative/3_1_fcci.tex]: I tried we can afford it :)

File: narrative/3_2_robustness.tex [hunk 9]
```diff
@@ -25,7 +25,7 @@ \subsection{Robustness assessment}
 example, a simple description of the enrichment facility (as a simple relation
 between the different assay with a SWU constrain) will be enough for the
 simulation of the history of a national fleet, but a fuel cycle evaluation of
-the JCPOA agreement will require a more detailed model of the enrichment
+the JCPOA agreement will require a more detailed model of the enrichment%JCPOA reference ?
```
Comment (gonuke) [inline narrative/3_2_robustness.tex]: Probably, since we don't define JCPOA anywhere

**Comment by gonuke** [comment]
Does this still build to be only 10 pages?
If you add a JCPOA reference, I think we're good to go!

**Comment by bam241** [comment]
yes still 10 pages. (without JCPOA reference)

File: narrative/3_2_robustness.tex [hunk 10]
```diff
@@ -102,7 +102,7 @@ \subsubsection{Analysis of Robustness to Input Parameters}
 in both modeling choice and input parameters, to determine if there are
 interactions between them.  That is, are there circumstances where the
 modeling choice makes the simulation more, or less, sensitive to variations in
-the input parameters.
+the input parameters?
```
Comment (bam241) [inline narrative/3_2_robustness.tex:105]: could you confirm this correction is correct ?

File: narrative/3_2_robustness.tex [hunk 11]
```diff
@@ -55,7 +55,7 @@ \subsubsection{Experiment Definition}
 nuclear fuel cycle applications will be considered, including historical
 records of US (large heterogeneous fleet) and/or French (fleet undergoing
 transition to plutonium recycling)\cite{courtin.phd} commercial
-nuclear fleets and EG-like transition\cite{Bo.ANSW.2016}.
+nuclear fleets and EG-like transitions\cite{Bo.ANSW.2016}.
```
Comment (bam241) [inline narrative/3_2_robustness.tex:58]: this one as well

**Comment by bam241** [comment]
still 10 pages with jcpoa ref

File: narrative/3_2_robustness.tex [hunk 12]
```diff
@@ -16,7 +16,7 @@ \subsection{Robustness assessment}
 comparisons are difficult. As with many simulation communities, there is a
 natural tendency to steadily improve the modeling fidelity.  However, some
 experts argue that the uncertainty in the input data for a prospective
-scenario result in output uncertainties that render increased fidelity
+scenario results in output uncertainties that render increased fidelity
```
Comment (bam241) [inline narrative/3_2_robustness.tex:19]: and this too

**Comment by gonuke** [comment]
Super - thanks so much!

**Pull request merged**